mining/mining.go

```diff
@@ -602,7 +602,7 @@ mempoolLoop:
 	blockSigOpCost := coinbaseSigOpCost
 	totalFees := int64(0)
 
-	// Query the version bits state to see if segwit has been activated, if
+	// Query the version bits state to see whether segwit has been activated, if
```

Comment:
I don't think there is any need for this change.  The simpler original version is fine.

Reply:
Thanks for your opinion and i will discard this change.